cyclestreets.app/src/main/assets/whatsnew.html

```diff
@@ -8,13 +8,17 @@
 <h3>Release 3.9 (BETA)</h3>
 <strong>Features:</strong>
 <ul>
-  <li>Coming soon...</li>
+  <li>POI URLs (e.g. bike shop websites) now appear as clickable links.</li>
```

Comment:
FYI @HilaryN I pushed an extra commit here to update the What's New (both the in-app page, this file, and the Beta download store listing, in `beta.txt`, to describe the new functionality.